### DISCUSSION TITLE
How do I run the bootstrap command?

### DISCUSSION THREAD

**Discussion by krystalcode** [postalserver/postal#2370] [Help with using Postal] [answered]
I am trying to install Postal in a pod using podman. I run the container from the registry i.e. `ghcr.io/postalserver/postal:2.1.2`. I enter the container and I run the `postal` command. There is no `bootstrap` sub-command and I therefore cannot initialize the configuration from within the container.

```
Usage: postal [command]

Server components:

 * web-server - run the web server
 * smtp-server - run the SMTP server
 * worker - run a worker
 * cron - run the cron process
 * requeuer - run the message requeuer

Setup/upgrade tools:

 * initialize - create and load the DB schema
 * upgrade - upgrade the DB schema

Other tools:

 * version - show the current Postal version
 * make-user - create a new global admin user
 * default-dkim-record - display the default DKIM record
 * console - open an interactive console
 * test-app-smtp - send a test message through Postal
```

**Comment by willpower232** [comment, marked as answer]
`postal bootstrap` is from the install binary, not the binary inside the container

https://github.com/postalserver/install/blob/main/bin/postal

vs

https://github.com/postalserver/postal/blob/main/bin/postal

If you're running the containers directly then you may want to read the install repository for additional context.
